MACHINE_LEARNING/machine_learning/models/resnet.py

```diff
@@ -16,15 +16,13 @@
 import numpy as np
 
 
-# noinspection SpellCheckingInspection
 class ResNet:
     """Modelo ResNet50"""
     def __init__(self):
         # Definir el modelo preentrenado
         self.name = 'ResNet50'
         self.model = ResNet50(weights='imagenet', include_top=True)
 
-    # noinspection SpellCheckingInspection
     # Método que realiza la predicción de objetos en una imagen
```

Comment:
Please translate to english from here to the end

Reply:
Solved! Thank you!